wire-ios/Wire-iOS/Sources/UserInterface/ConversationList/Container/ConversationListViewController.swift

```diff
@@ -34,6 +34,8 @@ final class ConversationListViewController: UIViewController {
     /// internal View Model
     var state: ConversationListState = .conversationList
 
+    private var previouslySelectedTabIndex = 1
```

Comment:
Why is it initialised with `1`?

Reply:
The tabs currently are [Contacts, Conversations, Folders, Archive] and the default tab is the second.